conda.recipe/meta.yaml

```
@@ -0,0 +1,48 @@
+# load information from pyproject.toml
+{% set pyproject = load_file_data('pyproject.toml') %}
+{% set project = pyproject.get('project', {}) %}
+{% set license = project.get('license').get('text') %}
+{% set description = project.get('description') %}
+{% set project_url = pyproject.get('project', {}).get('urls') %}
+{% set url = project_url.get('homepage') %}
+# this will get the version set by environment variable
+{% set version = environ.get('VERSION') %}
+{% set version_number = version.split('+')[0] %}
+{% set git_describe_number = environ.get('GIT_DESCRIBE_NUMBER', '0') | string %}
+
+package:
+  name: finddata
+  version: {{ version_number }}
+
+source:
+  path: ..
+
+build:
+  noarch: python
+  number: {{ git_describe_number }}
+  string: py{{py}}
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+
+requirements:
+  host:
+    - python
+    - versioningit
```

```suggestion
    - versioningit
    - setuptools
    - wheel
```
We  had to include this in drtsans' conda recipe, and it's now in the python_project_template repo.